### PR TITLE
Devil Reef encounter cards

### PR DESCRIPTION
**Pull request by grabben2**

### CODE REVIEW

File: pack/tic/def_encounter.json [hunk 1]
```diff
@@ -0,0 +1,598 @@
...
+		"pack_code": "def",
+		"position": 177,
+		"quantity": 1,
+		"shroud": 5,
+		"text": "<b>Forced</b> - After this location is revealed: Place the set-aside black key on it.\n[free] If investigators at this location control both the red key and the green key and spend 2 [per_investigator] clues, as a group: Take control of the black key and read <b>Flashback XI</b> in the Campaign Guide.",
+		"traits": "Cave. Y'ha-nthlei.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"They had all kinds o' cities on the sea-bottom, an' this island was heaved up from thar.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:398]: Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).

File: pack/tic/def_encounter.json [hunk 2]
```diff
@@ -0,0 +1,598 @@
...
+		"pack_code": "def",
+		"position": 176,
+		"quantity": 1,
+		"shroud": 1,
+		"text": "<b>Forced</b> - After this location is revealed: It becomes fully flooded. Place the set-aside white key on it.\n[free] If investigators at this location control both the blue key and the green key and spend 3 [per_investigator] clues, as a group: Take control of the white key and read <b>Flashback X</b> in the Campaign Guide.",
+		"traits": "Cave. Y'ha-nthlei.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"They had all kinds o' cities on the sea-bottom, an' this island was heaved up from thar.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:378]: Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).

File: pack/tic/def_encounter.json [hunk 3]
```diff
@@ -0,0 +1,598 @@
...
+		"pack_code": "def",
+		"position": 176,
+		"quantity": 1,
+		"shroud": 3,
+		"text": "<b>Forced</b> - After this location is revealed: It becomes fully flooded. Place the set-aside white key on it.\n[free] If investigators at this location control both the red key and the yellow key and spend 2 [per_investigator] clues, as a group: Take control of the white key and read <b>Flashback X</b> in the Campaign Guide.",
+		"traits": "Cave. Y'ha-nthlei.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"They had all kinds o' cities on the sea-bottom, an' this island was heaved up from thar.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:358]: Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).

File: pack/tic/def_encounter.json [hunk 4]
```diff
@@ -0,0 +1,598 @@
...
+		"pack_code": "def",
+		"position": 175,
+		"quantity": 1,
+		"shroud": 2,
+		"text": "<b>Forced</b> - After this location is revealed: Increase its flood level. Place the set-aside purple key on it.\n[free] If investigators at this location control both the green key and the yellow key and spend 3 [per_investigator] clues, as a group: Take control of the purple key and read <b>Flashback IX</b>in the Campaign Guide.",
+		"traits": "Cave. Y'ha-nthlei.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"They had all kinds o' cities on the sea-bottom, an' this island was heaved up from thar.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:338]: Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).

File: pack/tic/def_encounter.json [hunk 5]
```diff
@@ -0,0 +1,598 @@
...
+		"pack_code": "def",
+		"position": 175,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "<b>Forced</b> - After this location is revealed: Increase its flood level. Place the set-aside purple key on it.\n[free] If investigators at this location control both the red key and the blue key and spend 2 [per_investigator] clues, as a group: Take control of the purple key and read <b>Flashback IX</b> in the Campaign Guide.",
+		"traits": "Cave. Y'ha-nthlei.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"They had all kinds o' cities on the sea-bottom, an' this island was heaved up from thar.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:318]: Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).

File: pack/tic/def_encounter.json [hunk 6]
```diff
@@ -0,0 +1,598 @@
...
+		"position": 174,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "<b>Forced</b> - After Bootlegger's Hideaway is revealed: Randomly choose 1 of the set-aside facedown keys and place it on Bootlegger's Hideaway without looking at it.\n[action] If you control the black key: Gain 5 resources. (Group limit once per game.)",
+		"traits": "Cave.",
+		"type_code": "location",
+		"victory": 1
+	},
+	{
+		"back_flavor": "\"They had all kinds o' cities on the sea-bottom, an' this island was heaved up from thar.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:298]: Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).

File: pack/tic/def_encounter.json [hunk 7]
```diff
@@ -0,0 +1,598 @@
...
+		"encounter_code": "devil_reef",
+		"encounter_position": 15,
+		"faction_code": "mythos",
+		"illustrator": "Andrew Olson",
+		"name": "Deep One Grotto",
+		"pack_code": "def",
+		"position": 175,
+		"quantity": 1,
+		"shroud": 2,
+		"text": "<b>Forced</b> - After this location is revealed: Increase its flood level. Place the set-aside purple key on it.\n[free] If investigators at this location control both the green key and the yellow key and spend 3 [per_investigator] clues, as a group: Take control of the purple key and read <b>Flashback IX</b>in the Campaign Guide.",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:333]: Please add a space after Flashback IX's bold tag.

File: pack/tic/def_encounter.json [hunk 8]
```diff
@@ -0,0 +1,598 @@
...
+	{
+		"back_flavor": "The air is frigid here, and there is no sound but the faint plinking of water dripping onto the rough stone ground. You shudder and proceed onward...",
+		"back_illustrator": "Patrick McEvoy",
+		"back_name": "Tidal Tunnel",
+		"clues": 1,
+		"code": "07174b",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 13,
+		"faction_code": "neutral",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:285]: Mythos, not neutral.

File: pack/tic/def_encounter.json [hunk 9]
```diff
@@ -0,0 +1,598 @@
...
+	{
+		"back_flavor": "The air is frigid here, and there is no sound but the faint plinking of water dripping onto the rough stone ground. You shudder and proceed onward...",
+		"back_illustrator": "Patrick McEvoy",
+		"back_name": "Tidal Tunnel",
+		"clues": 1,
+		"code": "07174a",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 12,
+		"faction_code": "neutral",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:264]: Mythos, not neutral.

File: pack/tic/def_encounter.json [hunk 10]
```diff
@@ -0,0 +1,598 @@
...
+		"pack_code": "def",
+		"position": 172,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "[action] If you control the purple key: Look at the revealed side of any Tidal Tunnels or Unfathomable Depths location in play. (Group limit once per game.)\n<b>Forced</b> - After you reveal Salt Marshes: Increase its flood level. Put 1 random set-aside Tidal Tunnel into play adjacent to Salt Marshes.",
+		"traits": "Ocean. Island.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"...as I glimpsed it I could not help thinking of all the hideous legends I had heard in the last twenty-four hours - legends which portrayed this ragged rock as a veritable gateway to realms of unfathomed horror and inconceivable abnormality.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:236]: Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).

File: pack/tic/def_encounter.json [hunk 11]
```diff
@@ -0,0 +1,598 @@
...
+		"pack_code": "def",
+		"position": 171,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "<b>Forced</b> - After you reveal Waveworn Island: Increase its flood level. Put 1 random set-aside Unfathomable Depths and 2 random set-aside Tidal Tunnels into play, each adjacent to Waveworn Island (but not to one another).",
+		"traits": "Ocean. Island.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"...as I glimpsed it I could not help thinking of all the hideous legends I had heard in the last twenty-four hours - legends which portrayed this ragged rock as a veritable gateway to realms of unfathomed horror and inconceivable abnormality.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:216]: Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).

File: pack/tic/def_encounter.json [hunk 12]
```diff
@@ -0,0 +1,598 @@
...
+		"pack_code": "def",
+		"position": 170,
+		"quantity": 1,
+		"shroud": 3,
+		"text": "<b>Forced</b> - After you reveal Hidden Cove: Put 1 set-aside Tidal Tunnel into play adjacent to Hidden Cove, then put 1 set-aside Unfathomable Depths into play adjacent to that Tidal Tunnel.",
+		"traits": "Ocean. Island.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"...as I glimpsed it I could not help thinking of all the hideous legends I had heard in the last twenty-four hours - legends which portrayed this ragged rock as a veritable gateway to realms of unfathomed horror and inconceivable abnormality.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:196]: Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).

File: pack/tic/def_encounter.json [hunk 13]
```diff
@@ -0,0 +1,598 @@
...
+		"pack_code": "def",
+		"position": 169,
+		"quantity": 1,
+		"shroud": 5,
+		"text": "<b>Forced</b> - After Lonely Isle is revealed: Put 2 random set-aside Tidal Tunnels into play, each adjacent to Lonely Isle (but not to one another).",
+		"traits": "Ocean. Island.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"...as I glimpsed it I could not help thinking of all the hideous legends I had heard in the last twenty-four hours - legends which portrayed this ragged rock as a veritable gateway to realms of unfathomed horror and inconceivable abnormality.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:176]: Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).

File: pack/tic/def_encounter.json [hunk 14]
```diff
@@ -0,0 +1,598 @@
...
+		"pack_code": "def",
+		"position": 168,
+		"quantity": 1,
+		"shroud": 6,
+		"text": "[action] If you are in the Fishing Vessel: <b>Resign.</b> Each investigator in the Fishing Vessel resigns. Remove Fishing Vessel from the game. Each resigning investigator places each of their purple, black, or white keys on the current act. All other investigators are left to fend for themselves.",
+		"traits": "Ocean.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"...as I glimpsed it I could not help thinking of all the hideous legends I had heard in the last twenty-four hours - legends which portrayed this ragged rock as a veritable gateway to realms of unfathomed horror and inconceivable abnormality.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:156]: Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).

File: pack/tic/def_encounter.json [hunk 15]
```diff
@@ -0,0 +1,598 @@
...
+		"pack_code": "def",
+		"position": 167,
+		"quantity": 1,
+		"stage": 1,
+		"text": "Keys can only be claimed from [[Y'ha-nthlei]] locations using the [free] ability on those locations.\n<b>Objective</b> - Find and claim as many of the following keys as you can from [[Y'ha-nthlei]] locations (purple, white, and black). If you control all 3 of these keys, advance.",
+		"type_code": "act"
+	},
+	{
+		"back_text": "Churning Waters enters play fully flooded <i>(even while it is unrevealed)</i>.",
+		"back_flavor": "\"And far out at sea, despite a high tide, I glimpsed a long, black line scarcely rising above the water yet carrying a suggestion of odd latent malignancy.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:138]: Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).

File: pack/tic/def_encounter.json [hunk 16]
```diff
@@ -0,0 +1,598 @@
...
+		"name": "The Devil of the Depths",
+		"pack_code": "def",
+		"position": 166,
+		"quantity": 1,
+		"stage": 2,
+		"text": "As an additional cost for you to move from one [[Ocean]] location to another while not in a vehicle, you must spend 2 actions.\n<b>Forced</b> - When your turn begins, if you are at a fully flooded location, you struggle for air: When your turn ends, if you did not enter an unflooded location, partially flooded location, or a vehicle during your turn, take 5 direct damage.",
+		"type_code": "agenda"
+	},
+	{
+		"back_text": "<b>(→R1)</b>",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:117]: Please can you move this line below 'back_name' for alphabetical purposes!

File: pack/tic/def_encounter.json [hunk 17]
```diff
@@ -0,0 +1,598 @@
...
+		"pack_code": "def",
+		"position": 165,
+		"quantity": 1,
+		"subname": "Writhing in the Deep",
+		"text": "<b>Spawn</b> - Fishing Vessel's location.\nHunter. Massive. Retaliate.\nThe Terror of Devil Reef cannot enter [[Cave]] locations.\n<b>Forced</b> - When The Terror of Devil Reef would be defeated: Instead, remove all damage from it and exhaust it. It cannot ready for the remainder of the round <i>(including the upkeep phase)</i>.",
+		"traits": "Monster. Elite.",
+		"type_code": "enemy"
+	},
+	{
+		"back_text": "Record in your Campaign Log <i>the tide has grown stronger</i>.\n<b>(→R1)</b>",
```
Comment (drawntotheflame) [inline pack/tic/def_encounter.json:97]: Please can you move this line below 'back_name' for alphabetical purposes!

**Review by drawntotheflame — COMMENTED** [review]
Great job - basically just tiny tweaks from me @grabben2

**Comment by grabben2** [comment]
Thanks @drawntotheflame, file updated with your suggestions.
Also, converted tab indents to spaces to match the editor config that @akaan introduced, I hope that's okay.

**Comment by drawntotheflame** [comment]
@Kamalisk this can be merged now - thank you.

**Pull request merged**